package.json

```diff
@@ -17,5 +17,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "jsonwebtoken": "^8.5.1"
```

Comment:
Nope, that should be in your `backend/package.json`. Which tends towards proving:
- you **once again** ran a `git add .`
- you didn't read what you're commiting
- your SM didn't read the whole thing
- @LuigiMenez  is getting tired!

Reply:
o github.com:WildCodeSchool/2022-03-JS-Lille-project-3-g2-wim.git
 ! [rejected]        fixconsole -> fixconsole (non-fast-forward)
error: failed to push some refs to 'git@github.com:WildCodeSchool/2022-03-JS-Lille-project-3-g2-wim.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.